code/modules/power/singularity/collector.dm

```diff
@@ -174,9 +174,9 @@
 	. = ..()
 	if(active)
 		if(!bitcoinmining)
-			to_chat(user, "<span class='notice'>[src]'s display states that it is processing <b>[DisplayPower(last_power)]</b>.</span>")
+			to_chat(user, "<span class='notice'>[src]'s display states that it has stored <b>[DisplayPower(stored_power)]</b>, and processing <b>[DisplayPower(min(stored_power, (stored_power*RAD_COLLECTOR_STORED_OUT)+1000))]</b>.</span>")
 		else
-			to_chat(user, "<span class='notice'>[src]'s display states that it is producing a total of <b>[last_power*RAD_COLLECTOR_MINING_CONVERSION_RATE]</b> research points per minute.</span>")
+			to_chat(user, "<span class='notice'>[src]'s display states that it has stored a total of <b>[stored_power*RAD_COLLECTOR_MINING_CONVERSION_RATE]</b>, and producing [(min(stored_power, (stored_power*RAD_COLLECTOR_STORED_OUT)+1000))*RAD_COLLECTOR_MINING_CONVERSION_RATE] research points per minute.</span>")

```

Comment:
Maybe create a function for this equation since it seems to be used in a few places